:sparkles: feat : 매거진 피드백(수정 요청) 등록 기능 구현

jiyoon0000 opened the pull request:
-MagazineFeedback 엔티티 및 Repository 추가
-피드백 작성 시 revisionCount 자동 증가
-피드백은 최대 3회까지 제한(FEEDBACK_LIMIT_EXCEEDED 예외 처리)
-Swagger 문서화 및 테스트용 http 파일 작성

jiyoon0000 committed:
:sparkles: feat : 매거진 피드백(수정 요청) 등록 기능 구현
-피드백 작성 시 revisionCount 자동 증가
-피드백은 최대 3회까지 제한(FEEDBACK_LIMIT_EXCEEDED 예외 처리)
-Swagger 문서화 및 테스트용 http 파일 작성

j-hann reviewed (approved):
굿굿👍🍀

jiyoon0000 committed:
:sparkles: feat: 매거진 피드백 등록 및 내 피드백 조회 구현
-본인이 남긴 피드백 목록 조회 기능 구현

sooyeoneo commented on src/main/java/com/salayo/locallifebackend/domain/magazine/feedback/dto/MagazineFeedbackCreateRequestDto.java:
```diff
@@ -0,0 +1,14 @@
+public class MagazineFeedbackCreateRequestDto {
+
+    @NotBlank(message = "피드백 내용은 필수입니다.")
+    private String content;
```
**수정X** 
`건의 사항`: 혹시 엄청 긴 피드백을 DB에 부하가 생길 가능성을 고려하여, 
만약을 대비한...피드백 내용 길이를 제한하는 방법은 어떠신가요? 🧑🏻‍💻💭

```    
@Size(max = 1000, message = "피드백 내용은 1000자 이내여야 합니다.") // 1000자, 1500자, 2000자 등등..
```

jiyoon0000 replied:
오? 그러네요! 바로 반영하겠습니당😎

sooyeoneo reviewed (approved):
깔끔 그 잡채 🖥️💗

jiyoon0000 committed:
:wrench: add : 권한 외 접근 시 피드백 조회 403 예외 발생하도록 URL 경로 수정
-Spring Security의 .requestMatchers("/localcreator/**").hasRole("LOCAL_CREATOR") 설정에 맞게 url 경로 수정

jiyoon0000 committed:
Merge branch 'develop' into feature/magazine-feedback

# Conflicts:
#	src/main/java/com/salayo/locallifebackend/global/error/ErrorCode.java

jiyoon0000 committed:
:sparkles: feat: MagazineFeedbackCreateRequestDto에 내용 길이 제한 추가
-@Size(max = 1000) 어노테이션 추가하여 피드백 내용 최대 1000자 제한
-과도한 입력에 대한 방어 및 DB 부하 방지 목적
